cmake/android/android_gradle_projects.cmake

```diff
@@ -82,6 +82,13 @@ foreach(fname ${GRADLE_WRAPPER_FILES})
   install(FILES "${OpenCV_SOURCE_DIR}/platforms/android/gradle-wrapper/${fname}" DESTINATION "${ANDROID_INSTALL_SAMPLES_DIR}/${__dir}" COMPONENT samples ${__permissions})
 endforeach()
 
+# set build.gradle namespace
+if(ANDROID_GRADLE_PLUGIN_VERSION VERSION_GREATER_EQUAL "7.3.0")
+  ocv_update(OPENCV_NAMESPACE_DECLARATION "namespace 'org.opencv'")
```

Comment:
`OPENCV_NAMESPACE_DECLARATION` -> `OPENCV_ANDROID_NAMESPACE_DECLARATION` is preferable to avoid possible conflicts.